src/plugins/explore/public/components/visualizations/gauge/gauge_vis_config.ts

```diff
@@ -15,16 +15,15 @@ export interface GaugeChartStyleControls {
   title: string;
   min?: number;
   max?: number;
-  baseColor: string;
-  thresholds: Threshold[];
```

Comment:
Instead of deleting them, let's change them to optional and mark them as `@deprecated`